# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/performance-timeline/idlharness.any.js.ini
# Commit: 5248abfed275
# Full Hash: 5248abfed275b2938ed64770bec58b3ff68d624b
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-12 21:56:33
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=mstange,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27008
# ==============================================================================

diff -r e17500911824 -r 5248abfed275 testing/web-platform/meta/performance-timeline/idlharness.any.js.ini
--- a/testing/web-platform/meta/performance-timeline/idlharness.any.js.ini	Fri Apr 12 15:49:22 2019 +0000
+++ b/testing/web-platform/meta/performance-timeline/idlharness.any.js.ini	Fri Apr 12 16:29:48 2019 +0000
@@ -2,7 +2,10 @@
   expected: TIMEOUT
 
 [idlharness.any.serviceworker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: attribute detail]
@@ -16,10 +19,13 @@
 
 
 [idlharness.any.sharedworker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]
@@ -30,10 +36,13 @@
 
 
 [idlharness.any.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]
@@ -44,10 +53,13 @@
 
 
 [idlharness.any.worker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]